src/UI/Component/Panel/Listing/Divider.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/* Copyright (c) 2017 Alex Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Listing divider
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Divider extends \ILIAS\UI\Component\Component {
```

Comment:
> Do we have this "only expand one group" behaviour there too? Where can I observe it, because I didn't, so far.

Exercises with multiple assignments. But these are more panels/subpanels than lists. But I think it would be feasible to have similar accordion behaviour regardless if it's a panel or a list structure. Accordions in learning modules also show this behaviour. But this would also not be a list context.

> Do we have this "multi level nesting" there too? I also did not observe it there till now.

The nesting of heading structures? We have this implicitly everywhere. Or do you mean something like item groups in item groups? The presentation of precondition items in the repository could be an example for this. But I would rather prefer to get rid of this kind of presentation.

> Until now, we also always somehow internalized the headings into the component

Yes, Headings are no separate concept for now. Also the Actions can be assigned to the outer component instead. But we have to deal with the Heading hierarchy somehow.

Would the following be ok for all of you
- We go with option 3
- No separate Heading component. We will keep trying to find a way to deal with the heading hierarchy.
- Groups will be in the Item namespace: Item > Group
- Listing accept an array of Item Groups only for now (if we really have the need to add single items here, we may do this in the future, but I think this is not the case).